### PR TITLE
Added POST toggle Admin in UsersController and UsersControllerTests

### LINKED ISSUE #8: endpoint POST /api/admin/users/toggleAdmin in the UsersController
There is an endpoint POST /api/admin/users/toggleAdmin in the UsersController to toggle the Admin status similar to [this one](https://github.com/ucsb-cs156/proj-gauchoride/blob/0774df1d094692ae429cd4e925901ffde3a3ade8/src/main/java/edu/ucsb/cs156/gauchoride/controllers/UsersController.java#L72C1-L83C1) from proj-gauchoride.

### PR DESCRIPTION
**Pull request by shuyiwan**
Added POST toggle Admin in UsersController and UsersControllerTests
Passed all maven tests and mutation tests
Also tested it is working on localhost.
Added "genericMessage" in API Controller.

image
Screenshot 2023-11-21 at 6 50 40 PM

Close #8

### CODE REVIEW

**Review by allychu — DISMISSED** [review]
LGTM! Good work!

File: pom.xml [hunk 1]
```diff
@@ -242,6 +242,7 @@
                     <verbose>true</verbose>
                     <targetClasses>
                         <param>edu.ucsb.cs156.*</param>
+                        <!-- <param>edu.ucsb.cs156.organic.controllers.UsersController</param> -->
```
Comment (pconrad) [inline pom.xml]: This is commented out code.  Please see: 
https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

**Review by pconrad — CHANGES_REQUESTED** [review]
Please see: https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

File: pom.xml (hunk 1 above)
Comment (shuyiwan) [reply pom.xml]: Thank you! I have removed the code.

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**